static/js/components/PodcastPlayButton_test.js

```diff
@@ -0,0 +1,75 @@
+// @flow
+import { assert } from "chai"
+
+import PodcastPlayButton from "./PodcastPlayButton"
+
+import IntegrationTestHelper from "../util/integration_test_helper"
+import { makePodcastEpisode } from "../factories/podcasts"
+import { wait } from "../lib/util"
+import { setAudioPlayerState, setCurrentlyPlayingAudio } from "../actions/audio"
+import { AUDIO_PLAYER_PAUSED, AUDIO_PLAYER_PLAYING } from "../lib/constants"
+
+describe("PodcastPlayButton", () => {
+  let helper, render, episode
+
+  beforeEach(() => {
+    helper = new IntegrationTestHelper()
+    episode = makePodcastEpisode()
+    render = helper.configureReduxQueryRenderer(PodcastPlayButton, { episode })
+  })
+
+  afterEach(() => {
+    helper.cleanup()
+  })
+
+  it("should initialize a podcast", async () => {
+    const { wrapper, store } = await render()
+    wrapper.find(".podcast-play-button").simulate("click")
+    await wait(100)
+
+    assert.deepEqual(store.getState().audio, {
+      playerState:      AUDIO_PLAYER_PLAYING,
+      currentlyPlaying: {
+        title:       episode.podcast_title,
+        description: episode.title,
+        url:         episode.url
+      }
+    })
+  })
+
+  it("should show play button if not playing, not initialized", async () => {
+    const { wrapper } = await render()
+    assert.equal(wrapper.find(".podcast-play-button").text(), "Playplay_arrow")
+  })
+
+  it("should show pause button when initialized, playing", async () => {
+    const { wrapper } = await render({}, [
```

Comment:
Should we make a helper function to initialize the player in this way, since it's used twice?

Reply:
oh yeah, I meant to clean that up earlier but forgot